tests/common/plugins/sanity_check/__init__.py

```diff
@@ -89,6 +89,7 @@ def sanity_check(testbed_devices, request):
 
     if skip_sanity:
         logger.info("Skip sanity check according to configuration of test script.")
+        yield
```

Comment:
@wangxin , to check. if i do not have this yield line, then it give me error.  like this one.

https://stackoverflow.com/questions/57068680/i-am-getting-fixture-error-when-running-python-selenium-scripts-from-command-lin

Reply:
Yes, `yield` is required for this code path. I missed it. 
Also checked the other two code paths that `return` without `yield`. They are OK because they have `pytest.fail`.